jobbergate-cli/jobbergate_cli/schemas.py

```diff
@@ -86,44 +86,66 @@ class JobbergateApplicationConfig(pydantic.BaseModel):
     application_config: Dict[str, Any]
 
 
+class TemplateFileResponse(pydantic.BaseModel, extra=pydantic.Extra.ignore):
+    filename: str
+    file_type: str
+    created_at: datetime
+    updated_at: datetime
+    url: str
```

Comment:
That is a good point, because the API is sending just the final path, not the entire URL. For instance `jobbergate/job-script-templates/1/upload/template/test-job-script.py.j2`. It aims to facilitate other services (UI, CLI, agent) to consume the files, but it is up to them to know the base path.

Should I rename the field to clarify it?

Reply:
ah, in that case yes. We could call it a path, but it's just a semantic difference.

Reply:
solved in [ebb655a](https://github.com/omnivector-solutions/jobbergate/pull/263/commits/ebb655a0a3a4606bb5af4f3efe7e11ca2ed073c5)